base/applications/utilman/umandlg/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+
+spec2def(UManDlg.dll umandlg.spec)
```

Comment:
Weird casing here?

Reply:
The library has UManDlg as the original filename in XP and Server 2003. I can change it to `umandlg.dll` if spec2def is case insensitive.

Reply:
yes please keep all lowercase letters.

Reply:
It is inconsistent with the rest of the CMakeLists